bedrock/products/templates/products/vpn/landing.html

```diff
@@ -54,11 +60,15 @@
       subheading=ftl('vpn-landing-hero-sub-heading-v2', fallback='vpn-landing-hero-sub-heading'),
       desc=ftl('vpn-landing-hero-desc')
     ) %}
-      <div class="js-vpn-scroll-to-pricing">
+    <div class="js-vpn-scroll-to-pricing">
+        {% block hero_cta_change %}
         <a class="mzp-c-button mzp-t-xl" href="#pricing" data-cta-type="button" data-cta-text="Scroll to pricing" data-cta-position="primary">
           {{ ftl('vpn-shared-subscribe-link') }}
         </a>
-
+        {% endblock %}
+        <p>
+          <a href="#pricing" data-cta-text="Scroll to pricing">View all pricing plans</a>
+        </p>
```

Comment:
This should be in your variation b template?

Reply:
👀💦